mpv-wrapper

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+ln -snf "$(xdg-user-dir PICTURES)" "$HOME/.var/app/io.mpv.Mpv/Pictures"
+mpv "$@"
```

Comment:
Try `exec mpv-bin "$@"` (after my suggstion above). The `exec` won't leave trailing shell process opened.